docs/docs/reference-dom-elements.md

```diff
@@ -29,31 +98,18 @@ select small source span strong style sub summary sup table tbody td textarea
 tfoot th thead time title tr track u ul var video wbr
 ```
 
-#### SVG elements
+## All Supported SVG Elements
```

Comment:
Ah OK - I did not see that those were different things. In that case I think this section should just not go into the doc, along with the React.DOM-specific stuff. Removed it